sys/include/net/ng_netdev.h

```diff
@@ -121,8 +121,8 @@ typedef struct {
      * @return              -EOVERFLOW if available space in *value* given in
      *                      *value_len* is too small to store the option value
      */
-    int (*get)(ng_netdev_t *dev, ng_netconf_opt_t opt,
-                      void *value, size_t *value_len);
```

Comment:
acutally I purposely put value and value_len in the same row, as they kind of belong together...